apps/camera/style/settings.css

```diff
@@ -153,6 +155,9 @@
   height: 9rem;
   margin-right: 5rem; /* 1 */
   line-height: 9rem;
+  white-space: nowrap; /* 2 */
+  text-overflow: ellipsis; /* 2 */
+  overflow: hidden; /* 2 */
```

Comment:
You already have text-overflow:ellipsis and overflow:hidden on the settings_title class below. Would it be more appropriate to just put the white-space:nowrap property settings_title instead of adding these three properties here?